### PR TITLE
feat: tuning and serving Gemma on Ray on Vertex AI

### PR DESCRIPTION
**Pull request by inardini**
<br>
This notebook demonstrates how to use Ray on Vertex AI for fine-tuning and serving Gemma on Vertex AI.
<br><br><br>

- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

**Comment by kittyabs** [comment]
Looks like "TensorBoard" wasn't fixed. It's still showing as "Tensorboard."

File: notebooks/official/ray_on_vertex_ai/get_started_with_gemma_rov.ipynb [hunk 1]
```diff
@@ -0,0 +1,2248 @@
+{
```
Comment (gericdong) [inline notebooks/official/ray_on_vertex_ai/get_started_with_gemma_rov.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.        --machine-type={E2_HIGHCPU_32} --timeout=3600 {build_path}
</pre><p>Should it be {BUILD_MACHINE_TYPE}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

**Comment by gericdong** [comment]
@inardini the integration test can't run to complete. Please check.

File: notebooks/official/ray_on_vertex_ai/get_started_with_gemma_rov.ipynb (hunk 1 above)
Comment (inardini) [reply notebooks/official/ray_on_vertex_ai/get_started_with_gemma_rov.ipynb:1]: done

**Comment by gericdong** [comment]
It's covered by another PR.

**Pull request closed without merging**